docs/sdks/sdk-ref-ios.md

```diff
@@ -0,0 +1,197 @@
+
+### Installing with CocoaPods
+
+Add the following entry in your `Podfile`:
+
+```
+pod 'UID2', '~> 1.2'
+```
+
+## Usage Guidelines
+
+The **UID2Manager** singleton is the primary developer API for the SDK for iOS. It is responsible for storing, refreshing, and retrieving the EUID Identity including the EUID token.
+
+For iOS, the `UID2Manager` is initialized automatically the first time it is accessed. You can configure it to support automatic or manual refresh capabilities.
+
+There are two ways to establish an initial EUID Identity:
+
+1. Generate the EUID identity using DII&#8212;email (hashed or unhashed) or phone number (hashed or unhashed). For integration instructions, see [Client-Side Integration Guide for Mobile](../guides/integration-mobile-client-side.md).
+
+2. Create an EUID identity from your server's back end and then pass it to the EUID SDK. For integration instructions, see [Client-Server Integration Guide for Mobile](../guides/integration-mobile-client-server).
+
+The EUID Mobile SDKs can perform refreshes of EUID identities, after an Identity is established. This is because the refresh functionality relies on the refresh tokens that are part of the EUID Identity.
+
+
+## Code Samples
+
+The following code samples provide examples of performing specific activities relating to managing EUID with the SDK for iOS.
+
+Generate an initial EUID Identity (for instructions, see [Client-Side Integration Guide for Mobile](../guides/integration-mobile-client-side#configure-the-uid2-mobile-sdk)):
+
+```js
+UID2Manager.shared.generateIdentity(
+    _ identity: IdentityType,
+    subscriptionID: String,
+    serverPublicKey: String,
+    appName: String? = nil
+)
+```
+Set the Initial EUID Identity (for instructions, see [Client-Server Integration Guide for Mobile](../guides/integration-mobile-client-server#configure-the-uid2-mobile-sdk)):
+
+``` javascript
+UID2Manager.shared.setIdentity(_ identity: UID2Identity)
+```
+
+Get the EUID token (advertising token) to pass to the Advertising SDK (for ad request or bidstream use):
+
+```js
+UID2Manager.shared.getAdvertisingToken()
+```
+
+## UID2Manager API
+
+This section includes the functions and variables that are part of the UID2Manager API.
+
+### Functions
+
+The following functions are available as part of the UID2Manager API:
+- [generateIdentity()](#generateidentity)
+- [setIdentity()](#setidentity)
+- [resetIdentity()](#resetidentity)
+- [refreshIdentity()](#refreshidentity)
+- [getAdvertisingToken()](#getadvertisingtoken)
+- [setAutomaticRefreshEnabled()](#setautomaticrefreshenabled)
+
+#### generateIdentity()
+
+Generate am EUID Identity using <Link href="../ref-info/glossary-uid#gl-dii">Directly identifying information (DII)</Link>. For details, see [Configure the EUID mobile SDK](../guides/integration-mobile-client-side.md#configure-the-uid2-mobile-sdk) in the *Client-Side Integration Guide for Mobile*.
```

Comment:
We might need to update the link ../guides/integration-mobile-client-side.md#configure-the-uid2-mobile-sdk to an EUID-specific one once it's created. Same as some other links.

Reply:
Ah yes thx. I missed that. For now, commented out all those links to a doc that isn't on EUID yet.